readers/influxdb/messages.go

```
@@ -140,7 +140,8 @@ func fmtCondition(chanID string, rpm readers.PageMetadata) string {
 	if err != nil {
 		return condition
 	}
-	json.Unmarshal(meta, &query)
+
+	_ = json.Unmarshal(meta, &query)
```

Do not ignore errors.